packages/bento-design-system/src/DateField/DateField.tsx

```diff
@@ -109,17 +124,35 @@ function SingleDateField({ disabled, readOnly, ...props }: Extract<Props, { type
           shortcuts={shortcuts}
         />
       )}
+    </>
+  );
+
+  return props.isStandalone ? (
```

Comment:
can't we just expose the `datePicker`  as a `DateInput` component and then define the `DateField` as another component wrapping the DateInput?

Reply:
That was my first approach, but the `useDatePicker` hook returns stuff that is necessary both for the _input_ and the _field_. If we separate them into two components, wouldn't we have to use `useDatePicker` both in `DateInput` and `DateField`?